docs/tutorials/01_introductory/viz_solar_system.py

```diff
@@ -91,35 +100,26 @@ def init_planet(filename):
 uranus_actor.SetScale(1, 1, 1)
 neptune_actor.SetScale(1, 1, 1)
 
-mercury_actor.SetPosition(7, 0, 0)
-venus_actor.SetPosition(9, 0, 0)
-earth_actor.SetPosition(11, 0, 0)
-mars_actor.SetPosition(13, 0, 0)
-jupiter_actor.SetPosition(16, 0, 0)
-saturn_actor.SetPosition(19, 0, 0)
-saturn_rings_actor.SetPosition(19, 0, 0)
-uranus_actor.SetPosition(22, 0, 0)
-neptune_actor.SetPosition(25, 0, 0)
+mercury_actor.SetPosition(r_mercury, 0, 0)
+venus_actor.SetPosition(r_venus, 0, 0)
+earth_actor.SetPosition(r_earth, 0, 0)
+mars_actor.SetPosition(r_mars, 0, 0)
+jupiter_actor.SetPosition(r_jupiter, 0, 0)
+saturn_actor.SetPosition(r_saturn, 0, 0)
+saturn_rings_actor.SetPosition(r_saturn, 0, 0)
+uranus_actor.SetPosition(r_uranus, 0, 0)
+neptune_actor.SetPosition(r_neptune, 0, 0)
```

Comment:
This is repetitive. An idea could be to do it during the actor creation.

same for `SetScale`